src/GitHub.Api/Git/GitClient.cs

```diff
@@ -391,7 +388,46 @@ public ITask<string> AddAll(IOutputProcessor<string> processor = null)
             return last;
         }
 
-        public ITask<string> Remove(IList<string> files,
+        public ITask<string> AddAll(IOutputProcessor<string> processor = null)
+        {
+            Logger.Trace("Add all files");
+
+            return new GitAddTask(cancellationToken, processor)
+                .Configure(processManager);
+        }
+
+		public ITask<string> Discard( IList<string> files,
+			IOutputProcessor<string> processor = null )
+		{
+			Logger.Trace("Checkout Files");
+
+			GitCheckoutTask last = null;
+			foreach( var batch in files.Spool( 5000 ) )
+			{
+				var current = new GitCheckoutTask( batch, cancellationToken, processor ).Configure( processManager );
+				if( last == null )
```

Comment:
There should be a space after `if`. There should be no spaces after `(` and before `)`